<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Overview</class>
 <widget class="QWidget" name="Overview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="graphArea" native="true"/>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>45</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Range:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="rangeCombo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>Last Week</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Last Two Weeks</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Last Month</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Last Two Months</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Last Three Months</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Last 6 Months</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Last Year</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Everything</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Custom</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="dateStartLabel">
        <property name="text">
         <string>Start:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="dateStart">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
        <property name="timeSpec">
         <enum>Qt::UTC</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="dateEndLabel">
        <property name="text">
         <string>End:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="dateEnd">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
        <property name="timeSpec">
         <enum>Qt::UTC</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton">
        <property name="toolTip">
         <string>Reset view to selected date range</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QToolButton {
 background: transparent;
 border-radius: 8px;
 border: 2px solid transparent;
}

QToolButton:hover {
 border: 2px solid #456789;
}

QToolButton:pressed {
 border: 2px solid #456789;
 background-color: #89abcd;
}</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="Resources.qrc">
          <normaloff>:/icons/refresh.png</normaloff>:/icons/refresh.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>4</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="dateLayout"/>
      </item>
      <item>
       <widget class="QToolButton" name="toggleVisibility">
        <property name="toolTip">
         <string>Toggle Graph Visibility</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QToolButton {
 background: transparent;
 border-radius: 8px;
 border: 2px solid transparent;
}

QToolButton:hover {
 border: 2px solid #456789;
}

QToolButton:pressed {
 border: 2px solid #456789;
 background-color: #89abcd;
}</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
        <property name="arrowType">
         <enum>Qt::DownArrow</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="graphCombo">
        <property name="toolTip">
         <string>Drop down to see list of graphs to switch on/off.</string>
        </property>
        <item>
         <property name="text">
          <string>Graphs</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
